src/structs.jl

```diff
@@ -127,6 +127,74 @@ consistency_config = ConsistencyConfig(
 """
 $(TYPEDEF)
 
+Parameters used by the biomass tests.
+
+# Fields
+$(TYPEDFIELDS)
+"""
+mutable struct BiomassConfig
+    biomass_strings::Vector{String}
+    atpm_strings::Vector{String}
+    growth_metabolites::Dict{String,String}
+    minimum_growth_rate::Float64
+    maximum_growth_rate::Float64
+    ignored_precursors::Vector{String}
+    essential_precursors::Dict{String,String}
+    optimizer_modifications::Vector{Function}
+end
+
+biomass_config = BiomassConfig(
```

Comment:
this might better be a `const` so people don't monkeypatch it, and perhaps call it `default_biomass_config` or so?

Reply:
actually, they should monkey patch part of it... this should stay so that people can fine tune their tests